src/Calcver/CommitInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Calcver {
+    public class CommitInfo
+    {
+        public string Id { get; set; }
+        public string Message { get; set; }
+        public string ShortId()
+            => Id.Substring(0, 7);
```

Comment:
Can you add a comment here to explain why we are doing this?

Reply:
this code was just moved from a different file, it will be addressed in another mr (providing configuration options for this value): https://github.com/PotatoEngineering/Calcver.Net/issues/6